tsconfig.build.json

```diff
@@ -0,0 +1,128 @@
+{
+  "files": [],
+  "references": [
+    {
+      "path": "./packages/@uppy/audio/tsconfig.build.json"
```

Comment:
I don't think we need this? We only need `tsconfig.json` at the root.

Reply:
tsconfig.json is used by editors like vscode and `yarn test:ts` in order to. tsconfig.build.json is used by `yarn build:ts` because we don't want to spit out for test files etc

Reply:
`yarn build:ts` worked for me before without new new file? What didn't it do?

Reply:
build:ts doesn't typecheck everything (for example *.test.ts files)